__tests__/endpoints.test.js

```diff
@@ -114,5 +114,40 @@ describe('GET /api/articles', () => {
         });
     })
  })
+})
 
+// comments
+//GET /api/articles/:article_id/comments
+describe('GET /api/articles/:article_id/comments', () => { 
+  test('should return array of comments for an article sorted by created_at DESC ', () => { 
+    return query(app)
+    .get('/api/articles/1/comments')
+    .expect(200)
+    .then((res) => {
+        const {comments} = res.body;
+        expect(comments.length).toBe(11);
+        comments.forEach(comment => {
+          expect(comments).toBeSortedBy('created_at',{descending: true})
+          expect(comment).toEqual(expect.objectContaining({
+            author: expect.any(String),
+            comment_id: expect.any(Number),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number)
+          })
+          )
+        });
+    })
+ })
```

Comment:
Just one happy path test missing here:
Some articles do not have any associated comments. When an article has no comments we would expect to get back a status 200 and an empty array []